mmv1/third_party/terraform/tests/resource_compute_forwarding_rule_test.go.erb

```diff
@@ -141,13 +168,36 @@ resource "google_compute_forwarding_rule" "foobar" {
   name        = "%s"
   port_range  = "80-81"
   target      = google_compute_target_pool.bar-tp.self_link
-<% unless version == 'ga' -%>
-  labels = {
-    "baz" = "qux"
+
+	<% unless version == 'ga' %>
+		labels = {
+			baz = "qux"
+		}
+		service_directory_registrations {
+			namespace = google_service_directory_namespace.examplens.namespace_id
+			service = google_service_directory_service.examplesvc.service_id
+		}
+		<% end %>
```

Comment:
nit: we use 2 spaces rather than tabs for spacing here.

Reply:
Replaced tabs with 2 spaces